c2corg_ui/static/js/style/autoheight.js

```diff
@@ -0,0 +1,66 @@
+goog.provide('app.autoHeightDirective');
+goog.provide('app.AutoHeightController');
+
+goog.require('app');
+
+
+/**
+ * Apply automatic height depending on text to all textarea tags
```

Comment:
Is it really what we want?
It makes sense for the description attribute but not for the summary that should remain... short.

In addition I find it really confusing to add an implicit change to a standard HTML element: it's then hard to figure out why standard textareas get larger with no apparent reason.

I would recommend to create a brand new attribute directive that could be explicitly be added to the textareas this features should actually be used for. Not to override the standard element as it's done a few lines below:
```
app.module.directive('textarea', app.autoHeightDirective);
```